deploy.sh

```diff
@@ -2,7 +2,7 @@
 
 new_tag="$1"
 
-yarn version --new-version $new_tag
+pnpm version $new_tag
```

Comment:
Does pnpm have a version command? This was the one thing that tripped me up when I tried to migrate this to pnpm before.

Reply:
I'll test this now, but if it doesn't, npm version should work.